eslint.config.js

```diff
@@ -20,6 +20,7 @@ export default defineConfig([
 			'@stylistic/function-call-argument-newline': ['error', 'consistent'],
 			'@stylistic/indent': ['error', 'tab'],
 			'@stylistic/multiline-comment-style': 'off',
+			'@stylistic/multiline-ternary': 'off',
```

Comment:
Suppose this check is turned off, as we do not control the formatting of the code?

Reply:
Correct. This rule was enabled before and enforced that all ternary expressions be multi-line consistently. I disabled it because short ternaries like `isFullscreen ? title : titleCancel` are perfectly readable on one line.